test/extended/openstack/cpms.go

```diff
@@ -65,8 +65,18 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack] ControlPlane Mach
 		cpmsSecurityGroups := make(map[string]struct{})
 		cpmsServerGroup := providerSpec.Get("value.serverGroupName").String()
 		cpmsSubnets := make(map[string]struct{})
-		var machineNovaAvailabilityZones []string
+		var cpmsNetworks []string
 		var cpmsNovaAvailabilityZones []string
+		cpmsFds := make([]map[string]string, 0)
+		machinesFds := make([]map[string]string, 0)
```

Comment:
Consider using slices of a custom type instead, to leverage strong typing (see my suggestion about `type FailureDomain` below)